File: Grandma/Grandma.py
exitParameter=False
goodbyeChecker= False
def function(value):
  global exitParameter
  global goodbyeChecker
  newList= list(value)
  lowerCaseChecker= False
  for x in newList:
    if (x.islower()):
      lowerCaseChecker=True
  if lowerCaseChecker == True:
        print("SPEAK UP, KID!")
  elif value=="": 
        print("WHAT?!")
  elif lowerCaseChecker == False and value != "GOODBYE!": 
        print("NO, NOT SINCE 1946!")
  if value=="GOODBYE!" and goodbyeChecker==False: 
        print("LEAVING SO SOON?")
        goodbyeChecker =True
  elif value== "GOODBYE!" and goodbyeChecker==True: 
        print("LATER, SKATER!") 
        exitParameter=True
  if exitParameter==True:
    return
  else:
    value=input()
    function(value)
  
value= input("HELLO KID\n")
function(value)

File: Grandma.js
let exitParameter=false 
let goodbyeChecker= false
function DeafGrandma(input){
    console.log(input)
    let isLowerCase=false
    str=input.replace(/[a-z]/g, "1")
    if (str.includes("1")){isLowerCase=true}
    if (input===""){alert("WHAT?!")}
    else if (isLowerCase === true){alert("SPEAK UP, KID!")}
    else if (isLowerCase === false && input !== "GOODBYE!") {alert("NO, NOT SINCE 1946!")}
    else if (input=== "GOODBYE!"&& goodbyeChecker===false) {alert("LEAVING SO SOON?"); 
    goodbyeChecker=true}
    else if (input=== "GOODBYE!" && goodbyeChecker===true) {alert("LATER, SKATER!");exitParameter=true}

    if (exitParameter===true){return}
    else {
        input=window.prompt()
        DeafGrandma(input)
    }

    
    }
 alert("HEY KID!")
 input=window.prompt()
 DeafGrandma(input)